app/models/middleware_server.rb

```diff
@@ -12,4 +12,22 @@ class MiddlewareServer < ApplicationRecord
   def metrics_capture
     @metrics_capture ||= ManageIQ::Providers::Hawkular::MiddlewareManager::LiveMetricsCapture.new(self)
   end
+
+  def tenant_identity
```

Comment:
This method is needed in the context of the miq control engine.
Classes that participate on alerting need to provide a default one.
This code was written some time ago and I can see now this code has been extracted into a mixin (I guess to reuse it in a similar way).
This logic is the default one for that.